.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release to Docker and git
+on:
+  push:
+    tags:
+       - v*
+#on: [push, pull_request]
+jobs:
+  
+  docker:
+    name: Release on Docker hub
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Check output
+        env:
+          RELEASE_VERSION: ${{ steps.vars.outputs.tag }}
+        run: |
+          echo $RELEASE_VERSION
+          echo ${{ steps.vars.outputs.tag }}
+      - uses: actions/checkout@v1
+      - name: Publish to Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
```

Comment:
There is no docker build command or other related action for build docker image ? Is it ok with this?